src/tests/unit/events/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Fortran tests
+
+add_executable(event_post_1 event_post_1.f90)
+target_link_libraries(event_post_1 OpenCoarrays)
+
+add_executable(event_post_2 event_post_2.f90)
```

Comment:
We should try to get in the habit of using more descriptive names. I very much appreciate the additional test, though.